include/oneapi/tbb/task_group.h

```diff
@@ -270,10 +272,10 @@ class task_group_context : no_copy {
 
     task_group_context(task_group_context* actual_context)
         : my_version{task_group_context_version::proxy_support}
+        , my_lifetime_state{lifetime_state::proxy}
```

Comment:
What about rename my_lifetime_state to my_state, because proxy doesn't look like something related to lifetime. Or other name.

Reply:
It is always a question of a meaningful name to chose. `my_state` is not an option as it is already [occupied ](https://github.com/oneapi-src/oneTBB/blob/3db67b5ba2a81bd1288325c5847e09e13c46f4d7/include/oneapi/tbb/task_group.h#L207):)

Reply:
But I think my_lifetime_state it's not good anymore

Reply:
Renamed to 'mt_state'